### ISSUE TITLE
SLF4J Error and Null Pointer Exception in LogStatGen.java

### ISSUE THREAD

**Issue by alishan2040** [logpai/AutoLog#4]
Hi, thanks for providing the implementation. I was facing an issue while running the tool on my system. The issue is given below: 
```java
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Start processing the log statement
Exception in thread "main" java.lang.NullPointerException
at LogStatGen.createCSVFile(LogStatGen.java:180)
at LogStatGen.main(LogStatGen.java:112)
```
The corresponding command I used: `java -jar LogStatGen.jar -j ../hadoop-mapreduce-client-jobclient-2.6.4.jar -o log_methods.txt -m 'log'`
Although the above command was unsuccessful, I can still see non-empty files (_labelling.csv, logMethods.json)

Moving to the step 4 ( for getting logEP for log-related methods): 
I faced another issue (shown below) with the command `java -jar LogEPGen.jar -j ../hadoop-mapreduce-client-jobclient-2.6.4.jar -l log_methods.txt -o log_file.json -m 'log'`: 
```usage: OptionsUsage
 -h,--help          Print usage
 -j,--jarName       Jar file to contain the Log Statements
 -l,--LogMethods    LogMethods to analyze
 -m,--matcher       Matcher for Log API
 -o,--Output        Output file
```

I understand that `log_methods.txt` should be produced after step 1 (but here the exception is not about the missing `log_methods.txt` file.

Thanks

**Comment by YichenLi00** [comment]
Issue received. Could please provide the corresponding file and jdk version to me? I will help with issue in coming days.

**Comment by alishan2040** [comment]
By corresponding file, did you mean the jar file I used? (if so I downloaded it from here: https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-mapreduce-client-jobclient/2.6.4/hadoop-mapreduce-client-jobclient-2.6.4.jar)

Here is the jdk version:
```
openjdk version "11.0.20.1" 2023-08-24
OpenJDK Runtime Environment (build 11.0.20.1+1-post-Ubuntu-0ubuntu122.04)
OpenJDK 64-Bit Server VM (build 11.0.20.1+1-post-Ubuntu-0ubuntu122.04, mixed mode, sharing)
```

If you need other output files (e.g., `_labelling.csv`, `logMethods.json`), let me know and I'll share the link to those files.

Thanks for looking into the issue.

**Comment by YichenLi00** [comment]
@alishan2040 
Cloud you please rerun the updated tool which just been merged in our new bug fixing PR? I hope the updated version can solve your problem. Looking forward to your update.

**Comment by YichenLi00** [comment]
Sorry I can not reproduce your bugs..  Could you please share the jar file you use? I will help you with that.

**Comment by alishan2040** [comment]
Hi, the jar file I used is taken from here: https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-mapreduce-client-jobclient/2.6.4/hadoop-mapreduce-client-jobclient-2.6.4.jar

**Comment by YichenLi00** [comment]
> Hi, the jar file I used is taken from here: https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-mapreduce-client-jobclient/2.6.4/hadoop-mapreduce-client-jobclient-2.6.4.jar

Working on it..

**Comment by YichenLi00** [comment]
Hi @alishan2040 , I still can not reproduce your issue. It seems a bug/path error with JavaCG, you can check the usage of this tool from https://github.com/gousiosg/java-callgraph. I hope this could help you well.
